api/database/migrations/20191031003446-profile-additional-fields.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+
+module.exports = {
```

Comment:
Added some new fields.. also modified existing fields to make sure we were consistent with `enum` values between client and server

Reply:
I support this